.github/workflows/ci.yml

```
@@ -0,0 +1,122 @@
+---
+name: ci
+
+'on':
+  pull_request:
+  push:
+    branches:
+      - master
```

```suggestion
    branches: [main, master]
```

This could do with being both main and master for when we switch over to main 👍🏼